packages/mock-block-dock/src/hook-portals/text.tsx

```diff
@@ -90,6 +99,10 @@ export const TextHookView = ({
     );
   }
 
+  const previousTextRef = useRef<MaybePlainOrRichText>(
+    generateComparableString(text),
+  );
```

Comment:
You don't want to call `generateComparableString` on every render – but also this ref will always contain a string not `MaybePlainOrRichText`. 

```suggestion
  const previousTextRef = useRef<string | null>(null);
  if (previousTextRef.current === null) {
    previousTextRef.current = generateComparableString(text)
  }
```